packages/react-i18n/src/i18n.ts

```diff
@@ -227,6 +262,19 @@ export default class I18n {
       });
     }
   }
+
+  private decimalSymbol(currencyCode: string) {
+    const {symbol} = this.getCurrencySymbolLocalized(this.locale, currencyCode);
+    const templatedInput = Number('1000.00');
```

Comment:
A bit of a clever-ish trick to get the decimal symbol for the respective locale. Ideally, we would want to use [`Intl.Numberformat.formatToParts()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat/formatToParts) but it's not supported on all majore browsers and the [polyfill](https://github.com/tc39/proposal-intl-formatToParts/blob/master/polyfill.js) seems to only provide support for `Intl.DateTimeFormat.formatToParts`